.github/workflows/deploy-cloud.yaml

```diff
@@ -34,32 +40,51 @@ jobs:
       - name: Build
         run: |
           npm run build
+
+      - name: Configure AWS Credentials
```

Comment:
two comments here from me:
 * this is primary candidate to be abstarcted in workflow call
 * we could stop duplicating these steps and use https://github.com/actions/runner/issues/409#issuecomment-727565588, but, I believe, my current approach is more readable.

Reply:
I agree with that, but it's outside of the scope of the ticket I'm working (and I know that you're last time pretty vocal about not doing additional work in ticket).

If that would be only SRE-maintained repository, I would probably try to refactor that, but as it's not maintained by SRE, I would prefer to leave that topic for @saleor/dashboard team to address.